xbmc/utils/GUIUtils.h

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (C) 2016-2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+
+#pragma once
+
+#include "FileItem.h"
+#include "filesystem/MusicDatabaseDirectory.h"
+#include "filesystem/VirtualDirectory.h"
+#include "music/MusicDatabase.h"
+#include "music/MusicDbUrl.h"
+#include "threads/Event.h"
+#include "utils/JobManager.h"
```

Comment:
some of the includes are not needed, some are missing. some can be replaced by forward declarations.